scene/gui/tab_container.cpp

```diff
@@ -293,6 +295,8 @@ void TabContainer::_notification(int p_what) {
 }
 
 int TabContainer::_get_tab_width(int p_index) const {
+
+	ERR_FAIL_COND_V(p_index >= _get_tabs().size(), 0);
```

Comment:
Actually I had given this example out of my head like pseudo code, but there's a better macro for that:
```
ERR_FAIL_INDEX_V(p_index, _get_tabs().size(), 0);
```

Reply:
Could you also squash all commits together? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase